docs/UserManuals/TeamConfiguration.md

```diff
@@ -52,19 +52,19 @@ b. The actual api request.
 
     iii. After successful execution, the users table is generated and the data can be seen in the database table users.
 
-![image](/img/Team/teamflow1.png)
+![image](../../static/img/Team/teamflow1.png)
     
     iv. Generated the team_users table, you can see the data in the team_users table.
 
-![image](/img/Team/teamflow2.png)
+![image](../../static/img/Team/teamflow2.png)
 
 ## Step 3 - Update users if you need  
 If there is a problem with team_users association or data in users, just re-put users api interface, i.e. (b in step 2 above)
 
 ## Step 4 - Collect accounts 
```

Comment:
@hezyin I think we need to be more clear here. This guide generates dummy data for demoing purposes, but this step is requiring actual data to be present in the accounts table. Even if we run some plugin execution, that's not going to be generating account data that would correspond to the dummy user data. As a result, the user (the person running the demo) technically has to go in the accounts table and manually add rows that'll correspond to the dummy users, in order to make steps 5-7 be able to fetch user-account associations. I think this is a gap we need to fill in the doc.

Reply:
Thank you, keon. Very good advice, I will update the documentation.

Reply:
updated: https://github.com/apache/incubator-devlake-website/pull/120/files